src/components/PaymentLog.vue

```diff
@@ -570,6 +573,9 @@ export default {
                     this.apiUrls.businessTypes
                 );
                 this.businessTypes = fetchBusinessTypes.data.data.data;
+                this.businessTypes = this.businessTypes.filter(item => {
+                  return item.name === "Altara Credit Cash Loan" || item.name === "Altara Credit Products"
```

Comment:
This is not the right way to do it, filter by if name has Altara Credit

Reply:
okk